shared/templates/create_sebool_oval.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python2
+
+import sys
+import csv
+import re
+import os
+
+
+# Define input template filename to resulting output filename mapping
+files = { 'template_sebool' : 'sebool_' }
+
+PENDING = '>SEBOOL_BOOL</linux:pending_status'
+CURRENT = '>SEBOOL_BOOL</linux:current_status'
+EXTERN_VAR = '''</linux:selinuxboolean_state>
+
+  <external_variable comment="external variable for %s"
+  datatype="boolean" id="var_%s" version="1" />
+''' 
+
+
+def bool_state(sebool_state):
+    sebool = ""
+
```

Comment:
Looks good to me in general. But I would suggest ```bool_state``` routine to have check if ```sebool_state``` has one of the allowed values ```["on", "off", "enable", "disable"]``` and exit with error if not. Because having a first (used) row of ```selinux_booleans.csv``` in the form of:
```
# Add <selinux_boolean_name, desired_value> to generate hard-coded OVAL content.
# Add <selinux_boolean_name, use_var> to generate OVAL content that use XCCDF values.
abrt_anon_write, off
...
```
the value of ```sebool_state``` will be ``` off```, thus subsequently ```bool_state()``` routine will keep the ```sebool``` value set to ```''``` (empty string), and produced ```sebool_abrt_anon_write.xml``` will have same output regardless if ``` off``` or ``` on``` was specified in the CSV file. In other words, if ```off```, ```on```, ```enable```, ```disable``` values are the only allowed ones, the script should complain when ``` off``` is used (IOW expected value but prefixed with space). Since this might lead to produced *.xml file to have incorrect content.